eclipsecs-sevntu-plugin/src/com/github/sevntu/checkstyle/checks/coding/checkstyle-metadata.properties

```diff
@@ -188,3 +188,10 @@ MoveVariableInsideIfCheck.desc = Checks if a variable is only used inside if sta
 
 RequireFailForTryCatchInJunitCheck.name = Require Fail For Try/Catch in JUnit Check
 RequireFailForTryCatchInJunitCheck.desc = Checks if a try/catch block has a junit fail assertion inside the try  for a junit method.
+
+Jsr305AnnotationsCheck.name=Jsr305AnnotationsCheck
+Jsr305AnnotationsCheck.desc=Checks method parameters and return values for the presence of @Nonnull, @Nullable, or @CheckForNull annotations.
+Jsr305AnnotationsCheck.packages=Sets the parent package for all classes that will be checked.
+Jsr305AnnotationsCheck.excludePackages=Packages excluded from checking. This can be useful if under the parent package set with "packages" there are subpackages which should not be checked.
+Jsr305AnnotationsCheck.allowOverridingReturnValue=Annotating return values "@CheckForNull" in overridden methods is flagged as an error. When setting this property to true, this error will be ignored (useful for upgrading).
```

Comment:
`this error` => `this violation` 
user define severity of violation not a Check.

Please fix this in javadoc also, grep all other code to avoid "error" "warning" words.

Reply:
Done.

Reply:
I still see 8 in stances of error word, please replace with violation